assets/js/base/components/text-input/index.js

```diff
@@ -20,8 +20,8 @@ const TextInput = ( {
 	screenReaderLabel,
 	disabled,
 	help,
-	value,
-	onChange,
+	value = '',
+	onChange = () => null,
```

Comment:
By using `value` and not setting onChange, the component will always be controlled but unusable, do we have a use case in which `disabled` might not solve it? because I warry a11y about this, for a component that can’t be interacted with and is not disabled

Reply:
Good point! I wasn't sure about this either but looked at the work currently being done in the `<QuantitySelector>` to keep it consistent. But thinking about it twice and looking at Gutenberg components as a reference, I think that's a good point. I opened a discussion here: https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1333#discussion_r364769781. Depending on what we agree I will remove this change from this PR or keep it.

Reply:
As discussed in #1333, I removed the `onChange` default prop and made it required (5d1231c54123c3f2462f033dc96b291e0c012df4). I will proceed merging this PR because it has been approved, but let me know if the last commit doesn't look, @senadir.